CGATPipelines/pipeline_bamstats/pipeline.ini

```diff
@@ -63,7 +57,7 @@ name=csvdb
 #database location 
 database=/ifs/projects/adam/pipeline_publication/csvdb
```

Comment:
I think we should store this information in `/ifs/mirror` or similar but not on a project specific folder

Reply:
I agree, will make changes and push again